ql/src/java/org/apache/hadoop/hive/ql/optimizer/calcite/Bug.java

```diff
@@ -29,6 +29,8 @@
  *
  */
 public final class Bug {
+  
+  public static final int CALCITE_VERSION = 33;
```

Comment:
Let's use specific JIRA ids instead of general version numbers.

Reply:
Done!